packages/config/src/layer2s/fuelv1.ts

```diff
@@ -114,4 +114,26 @@ export const fuelv1: Layer2 = {
     ],
     risks: [],
   },
+  milestones: [
+    {
+      name: 'Fuel v1 is live on Mainnet',
+      link: 'https://twitter.com/fuellabs_/status/1344707195250896899',
+      date: '2020-12-31T00:00:00Z',
+      description:
+        'First trustless Optimistic Rollup is live on Mainnet.',
+    },
+    {
+      name: 'Beta-1 introduced',
```

Comment:
I would explicitly state that these milestones are about fuel v2. I think we can still track them but this will be a different project/deployment (similarly to zksync 1 vs zksync 2)

Reply:
These milestones should be removed from Fuel v1